clang/docs/checkedc/verification-with-seahorn.md

```diff
@@ -0,0 +1,96 @@
+# Verification of Unchecked codes with Seahorn
+
+Our goal is to verify whether unchecked functions violate their bounds-safe interface or not. We use [Seahorn](https://seahorn.github.io) tool as a backend verifier. Seahorn is a software verification framework. It can verify user-supplied assumption and assertions, as well as a number of built-in safety properties. We replace Seahorn's front-end with Checked-c clang to be able to process the bounds information and automatically insert necessary assertions to generate proper verification conditions.

```

Comment:
Typo: assumption->assumptions.

Also, we don't hyphenate Checked C.  It is "Checked C", not "Checked-C".